src/python/pants/backend/native/tasks/native_compile.py

```diff
@@ -176,21 +178,22 @@ def _make_compile_request(self, versioned_target, dependencies, external_libs_pr
       include_dirs=include_dirs,
       sources=sources_and_headers,
       fatal_warnings=self._compile_settings.get_fatal_warnings_value_for_target(target),
+      compiler_option_sets_options=self._compile_settings.get_merged_compiler_options_for_target(target),
```

Comment:
At this point they will have been flattened, so you can probably just call this field `compiler_options`.